CMakeLists.txt

```diff
@@ -202,6 +202,7 @@ sbuild_get_env(SBUILD_PATH PATH)
 sbuild_get_env(SBUILD_CPATH CPATH)
 sbuild_get_env(SBUILD_LIBRARY_PATH LIBRARY_PATH)
 sbuild_get_env(SBUILD_LD_LIBRARY_PATH LD_LIBRARY_PATH)
+sbuild_get_env(SBUILD_DYLD_LIBRARY_PATH DYLD_LIBRARY_PATH)
```

Comment:
This gets the content as a Cmake variable. The variable seems not to be ever used. So I'd say this can be removed. (in fact, not a problem with this PR, but `SBUILD_PATH`, and `SBUILD_LD_LIBRARY_PATH` are not used anymore either).

```suggestion
```

Reply:
Sure